clients/go/cmd/zbctl/internal/commands/root.go

```diff
@@ -46,13 +46,15 @@ var clientCacheFlag string
 var rootCmd = &cobra.Command{
 	Use:   "zbctl",
 	Short: "zeebe command line interface",
-	Long: `zbctl is a command line interface designed to create and read resources inside zeebe broker.
-It is designed for regular maintenance jobs such as:
+	Long: `zbctl is a command line interface for Camunda Cloud designed to create and read resources inside a Zeebe broker.
```

Comment:
:wrench: I would omit this change, since the naming is likely to change soon.

```suggestion
	Long: `zbctl is a command line interface designed to create and read resources inside a Zeebe cluster.
```